$NetBSD: patch-aa,v 1.2 2006/07/27 09:03:28 rillig Exp $

Another self-written configure script that does everything wrong. This
one does not let the user choose the directory where py-numarray is
installed.

--- configure.orig	2005-06-25 03:38:43.000000000 +0200
+++ configure	2006-06-28 20:05:53.000000000 +0200
@@ -1935,7 +1935,6 @@ fi
 echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6
 
-/* print extra warning messages when compiling CVS */
 test "`echo ${SG_VERSION} || grep -i cvs`" && CFLAGS="$CFLAGS -g -O -W -Wall -DWITH_WARNINGS"
 
 # Checks for programs.
@@ -10014,8 +10013,8 @@ echo "$as_me: error: Library requirement
   fi
 
 
-# Extract the first word of "libart-config", so it can be a program name with args.
-set dummy libart-config; ac_word=$2
+# Extract the first word of "libart2-config", so it can be a program name with args.
+set dummy libart2-config; ac_word=$2
 echo "$as_me:$LINENO: checking for $ac_word" >&5
 echo $ECHO_N "checking for $ac_word... $ECHO_C" >&6
 if test "${ac_cv_prog_libart_config_found+set}" = set; then
@@ -10113,10 +10112,10 @@ fi
 
 
 if eval "test x$libart_found != xno && test x$libart_config_found != xno"; then
- LIBART_CFLAGS="`libart-config --cflags`"
+ LIBART_CFLAGS="`libart2-config --cflags`"
 
  CFLAGS="$CFLAGS $LIBART_CFLAGS -DWITH_LIBART"
- LIBART_LIBS="`libart-config --libs`"
+ LIBART_LIBS="`libart2-config --libs`"
 
  LIBS="$LIBS $LIBART_LIBS"
 fi
@@ -10337,10 +10336,7 @@ fi;
                   PY_CFLAGS="-I$PY_PREFIX/include/python$PY_VERSION"
 		  PY_LIBS="python$PY_VERSION"
 		  PY_LIB_LOC="$PY_EXEC_PREFIX/lib/python$PY_VERSION/"
-                  array_loc=`find $python_numeric_path -name arrayobject.h | sed s/arrayobject\.h//`
-                  if test x$array_loc = x; then
-                     array_loc=`find $PY_PREFIX/ -name arrayobject.h | sed s/arrayobject\.h//`
-                  fi
+		  array_loc="${PY_PREFIX}/include/python${PY_VERSION}/numarray"
                   if test -n $array_loc -a -d $array_loc; then
                      echo "$array_loc"
                      PY_CFLAGS="$PY_CFLAGS -I$array_loc -DWITH_NUMERIC_PYTHON"
@@ -10374,7 +10370,7 @@
 
 if eval "test x$array_loc = x"; then
  echo "Numerical python not found!"
- exit
+ exit 1
 fi
 
 UNAME=`uname -s -m -r -p`
@@ -10414,12 +10410,10 @@ scigraphica_icondir="${datadir}/pixmaps/
 
 
 cat >>confdefs.h <<_ACEOF
-#define VERSION_INFO "sg-$VERSION\\n$UNAME\\n\
-gtk-$GTK_VERSION  gtkextra-$GTK_EXTRA_VERSION  \\n\
-xml-$XML_VERSION  python-$PY_VVERSION  $array_loc\\n"\
-__DATE__ " " __TIME__ "\\n\
-Compiler flags:$COMPILER_FLAGS\\n\
-(C) 2000 Adrian E. Feiguin\\n"
+#define VERSION_INFO			\
+	"sg-$VERSION\\n"		\
+	"$UNAME\\n"			\
+	"(C) 2000 Adrian E. Feiguin\\n"
 
 _ACEOF
 
